lddwrap/main.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+"""Command-line tool entry point for lddwrap to determine shared libraries required by a program."""
+
+# This file is necessary so that we can specify the entry point for pex.
+
+import argparse
+import pathlib
+import sys
+from typing import List, Any, TextIO
+
+import lddwrap
+import pylddwrap_meta
+
+
+class Args:
+    """Represent parsed command-line arguments."""
+
+    def __init__(self, args: Any) -> None:
+        """Initialize with arguments parsed with ``argparse``."""
+        self.version = bool(args.version)
+        self.format = str(args.format)
+        self.path = pathlib.Path(args.path)
+
+
+def parse_args(sys_argv: List[str]) -> Args:
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--version",
+        help="Display the version and return immediately",
+        action='store_true')
+    parser.add_argument(
+        "--format",
+        help="Specify the output format. [NEED PATH]",
+        default='verbose',
+        choices=['verbose', 'json'])
+
+    parser.add_argument(
+        "path", help="Specify path to the binary", nargs='?', default='')
+
+    args = parser.parse_args(sys_argv[1:])
+
+    if not args.version and not pathlib.Path(args.path).is_file():
+        parser.error(
+            "Either path '{}' is not given or is invalid. Check out --help for more information."
+            .format(args.path))
+
+    return Args(args=args)
+
+
+def _main(args: Args, stream: TextIO) -> int:
+    """Execute the main routine."""
+    # pylint: disable=protected-access
+    if args.version:
+        stream.write("pylddwrap version {}\n".format(
```

Comment:
Please remove the prefix "pylddwrap version " and output only the version:

````
stream.write(pylddwrap_meta.__version__ + "\n")
```

so that we can use the output of the version readily in other tools (*e.g.* when we need to decide whether a new version of lddwrap needs to be downloaded *etc.*).